<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transparencyPalette</class>
 <widget class="QWidget" name="transparencyPalette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="editLineSelector">
       <property name="toolTip">
        <string>Edit Line Color Properties</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editFillSelector">
       <property name="toolTip">
        <string>Edit Fill Color Properties</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>2</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageS">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ScrSpinBox" name="strokeOpacity">
           <property name="toolTip">
            <string>Set the transparency for the color selected</string>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Blend Mode:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="blendModeStroke">
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Darken</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lighten</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Multiply</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Overlay</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hard Light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Soft Light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Difference</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Exclusion</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color Dodge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color Burn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Saturation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Luminosity</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>240</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabSolid">
          <attribute name="title">
           <string>Solid</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Opacity:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ScrSpinBox" name="fillOpacity">
               <property name="toolTip">
                <string>Set the transparency for the color selected</string>
               </property>
               <property name="suffix">
                <string> %</string>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>195</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabGradient">
          <attribute name="title">
           <string>Gradient</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="gradientType">
               <item>
                <property name="text">
                 <string>Linear</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Radial</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="gradEditButton">
               <property name="toolTip">
                <string>Move the start of the gradient vector with the left mouse button pressed and move the end of the gradient vector with the right mouse button pressed</string>
               </property>
               <property name="text">
                <string>Move Vector</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="GradientEditor" name="gradEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>173</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="namedGradient"/>
           </item>
           <item>
            <widget class="QCheckBox" name="transpCalcGradient">
             <property name="text">
              <string>Use Luminosity as Alpha</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabPattern">
          <attribute name="title">
           <string>Pattern</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="patternBox">
             <property name="movement">
              <enum>QListView::Snap</enum>
             </property>
             <property name="flow">
              <enum>QListView::LeftToRight</enum>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>true</bool>
             </property>
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
             <property name="viewMode">
              <enum>QListView::IconMode</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QToolButton" name="editPatternProps">
               <property name="text">
                <string>Properties...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="transpCalcPattern">
             <property name="text">
              <string>Use Luminosity as Alpha</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="usePatternInverted">
             <property name="text">
              <string>Use inverted Pattern</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Blend Mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="blendModeFill">
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Darken</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lighten</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Multiply</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Screen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Overlay</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hard Light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Soft Light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Difference</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Exclusion</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color Dodge</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color Burn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Saturation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Luminosity</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>GradientEditor</class>
   <extends>QFrame</extends>
   <header>ui/gradienteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
